docs/DEVELOPMENT.md

```diff
@@ -38,6 +38,11 @@ To run the one or more tests via wildcard:
 ./scripts/tests.sh -run "Functional/pull*"
 ```
 
+To run tests in one specific package:
+```
+go test internal/pkg/PACKAGE/...
```

Comment:
Tebe to ide aj takto? mne to musi zacinat s `go test ./`, mozno linux specific, mozes to prosim upravit?

Reply:
nejde, máš pravdu, opravím 🙂